test/clone.test.js

```diff
@@ -0,0 +1,18 @@
+const assert = require('assert');
+const math = require('mathjs');
+
+const clone = require('../lib/util/clone');
+
+function testCloner(expNode, new_node) {
+  it ('creates a clone of a node', function () {
+    assert.equal(clone(math.parse(expNode)), new_node);
```

Comment:
Ok but one thing .. For eg - 

Node = `math.parse('x')  // Node { name: 'x', comment: '' }`

Node_To_Be_Clone = `clone(math.parse('x') // Node { name: 'x', changeGroup: undefined } `

And the test case that I made is that the value of the node that is being clone should be equal to the original one..  

Sounds good ? or you suggest !